syncer/syncer_test.go

```diff
@@ -951,8 +951,7 @@ func (s *testSyncerSuite) TestGeneratedColumn(c *C) {
 	syncer.toDBConns = []*DBConn{{baseConn: conn.NewBaseConn(dbConn, &retry.FiniteRetryStrategy{})}}
 	syncer.reset()
 
-	streamer, err := syncer.streamerProducer.generateStreamer(pos)
-	c.Assert(err, IsNil)
+	syncer.streamerController, err = NewStreamerController(*tcontext.Background(), syncer.syncCfg, syncer.fromDB, syncer.binlogType, syncer.cfg.RelayDir, syncer.timezone, pos)
```

Comment:
Shall we add `c.Assert(err, IsNil)` here?

Reply:
fix in e9e4868